packages/web-app-external/src/App.vue

```diff
@@ -36,9 +36,9 @@
 import { mapGetters } from 'vuex'
 import ErrorScreen from './components/ErrorScreen.vue'
 import LoadingScreen from './components/LoadingScreen.vue'
-import { computed, unref } from '@vue/composition-api'
+import { computed, unref } from 'vue'
 import { queryItemAsString, useAppDefaults, useRouteQuery } from 'web-pkg/src/composables'
-import { defineComponent } from '@vue/composition-api'
+import { defineComponent } from 'vue'
```

Comment:
Could be united with line 39, no? 🥷  Not worth an individual commit though, was just browsing and spotted this. Awesome work btw!